src/components/Checkbox/__stories__/checkbox.stories.mdx

```diff
@@ -0,0 +1,168 @@
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import Checkbox from "../Checkbox";
+import { createComponentTemplate } from "../../../storybook/functions/create-component-story";
+import { RADIO_BUTTON, TOGGLE, DROPDAWN } from "../../../storybook/components/related-components/component-description-map";
+import DialogContentContainer from "../../DialogContentContainer/DialogContentContainer";
+import { Link } from "../../../storybook/components";
+import "./checkbox.stories.scss";
+
+<Meta
+  title="Components/Checkbox"
+  component={ Checkbox }
+/>
+
+<!--- Component template -->
+
+export const checkboxTemplate = createComponentTemplate(Checkbox);
+
+<!--- Component documentation -->
+
+# Checkbox
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Checkboxes allow users to select one or more items from a set of options.
+<Canvas>
+  <Story name="Overview"
+         args={ {label: "Option", checked: true} }>
+    { checkboxTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ Checkbox } />
+
+## Usage
+<UsageGuidelines guidelines={[
```

Comment:
The way the arrow looks in the middle of the multiple text little bit not readable
<img width="453" alt="Screen Shot 2021-11-18 at 9 20 35" src="https://user-images.githubusercontent.com/72390374/142370186-a91173ec-3c48-470b-9e19-b1f00ab0688b.png">

Is it possible to use css alone to place the arrow near the first row and not in the middle of the element?

Reply:
We have no posibilities to style arrow outside 'UsageGuidlines'. We can change it inside component only.

Reply:
we will fix it in our side than :)